src/development/platform-integration/ios/platform-views.md

```diff
@@ -0,0 +1,434 @@
+---
+title: Hosting native iOS views in your Flutter app with Platform Views
+short-title: iOS platform-views
+description: Learn how to host native iOS views in your Flutter app with Platform Views.
+---
+
+<?code-excerpt path-base="development/platform_integration"?>
+
+Platform views allow you to embed native views in a Flutter app,
+so you can apply transforms, clips, and opacity to the native view
+from Dart.
+
+This allows you, for example, to use the native
+Google Maps from the Android and iOS SDKs
+directly inside your Flutter app.
+
+{{site.alert.note}}
+  This page discusses how to host your own native views
+  within a Flutter app.
+  If you'd like to embed native Android views
+  in your Flutter app,
+  see [Hosting native Android views][].
+{{site.alert.end}}
+
+[Hosting native Android views]: {{site.url}}/development/platform-integration/android/platform-views
+
+
+iOS only uses Hybrid composition,
```

Comment:
suggestion: "iOS uses only hybrid composition"

Also, hybrid composition doesn't appear to be defined on this page before this mention. Can we link to the definition, or should we lose the use of this undefined term?